.changeset/long-forks-bow.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': patch
+'@firebase/storage': patch
+---
+
+Fix infinite recursion in FirebaseStorageError message getter
```

Comment:
Is there an associated Github issue? Or an example of when the problem comes up in usage? So I can better explain to users in the release notes.

Reply:
No there isn't an existing issue (beyond this PR). Basically any time the `message` getter is accessed you would get a `Maximum call stack size exceeded` Error raised.